app/models/setting/template_sync.rb

```diff
@@ -7,21 +7,31 @@ def self.associate_types
     }
   end
 
+  def self.metadata_export_mode_types
+    {
+      'refresh' => _('Refresh'),
+      'keep' => _('Keep'),
+      'remove' => _('Remove')
+    }
+  end
+
+
   def self.load_defaults
     return unless super
 
-    %w(template_sync_filter template_sync_branch).each { |s| Setting::BLANK_ATTRS << s }
+    %w(template_sync_filter template_sync_branch ).each { |s| Setting::BLANK_ATTRS << s }
 
     self.transaction do
       [
         self.set('template_sync_verbose', N_('Choose verbosity for Rake task importing templates'), false, N_('Verbosity')),
         self.set('template_sync_associate', N_('Associate templates to OS'), 'new', N_('Associate'), nil, { :collection => Proc.new { self.associate_types } }),
-        self.set('template_sync_prefix', N_('The string all imported templates should begin with'), "Community ", N_('Prefix')),
+        self.set('template_sync_prefix', N_('The string all imported or exported templates should begin with'), "Community ", N_('Prefix')),
         self.set('template_sync_dirname', N_('The directory within the Git repo containing the templates'), '/', N_('Dirname')),
-        self.set('template_sync_filter', N_('Import names matching this regex (case-insensitive; snippets are not filtered)'), nil, N_('Filter')),
+        self.set('template_sync_filter', N_('Import or export names matching this regex (case-insensitive; snippets are not filtered)'), nil, N_('Filter')),
         self.set('template_sync_repo', N_('Default Git repo to sync from'), 'https://github.com/theforeman/community-templates.git', N_('Repo')),
-        self.set('template_sync_negate', N_('Negate the search'), false, N_('Negate')),
-        self.set('template_sync_branch', N_('Default branch in Git repo'), nil, N_('Branch'))
+        self.set('template_sync_negate', N_('Negate the prefix search for purging, filter for importing or exporting'), false, N_('Negate')),
```

Comment:
Nitpick: This can probably be worded better - perhaps "Negate the prefix (for purging) / filter (for importing/exporting)" ?